libs/ui/packages/icons/scripts/buildAdditionalIcons.js

```diff
@@ -0,0 +1,100 @@
+const fs = require("fs");
```

Comment:
Are you sure you need a separate script? At the very least I see a lot of duplicated code between the two scripts which could be externalized in a separate file imported by both.

Reply:
It's planned to merge those scripts, Initially for this task those icons were in separate packages but we decided to regroup everything in the Icons package. With not much time left I didn't had time to merge the scripts.

Reply:
> It's planned to merge those scripts, Initially for this task those icons were in separate packages but we decided to regroup everything in the Icons package.

Ah I see 🙇 

> With not much time left I didn't had time to merge the scripts.

Not sure I get it, are you planning to merge these scripts then? What do you mean by `not much time left` ?